### ISSUE TITLE
How to approve/reject user annotation?

### ISSUE THREAD

**Issue by nguyenvanhieuvn** [doccano/doccano#1335]
Hi, thanks for a great annotation tool.

I have one question, searching throung existing isues dont help me.
I have a project type **Sequence to sequence** have one user role Annotator and one as Annotation Approver.
When i'm login with Annotation Approver user account, it's seem everything like Annotator account. There's nothing to approve/reject an annotation.
- User manager page:
![Screenshot from 2021-04-27 10-19-48](https://user-images.githubusercontent.com/12376486/116179835-32639800-a742-11eb-87a2-a2e85c2755a4.png)

- Annotation page for Annotation Approver user:
![Screenshot from 2021-04-27 10-20-10](https://user-images.githubusercontent.com/12376486/116179852-368fb580-a742-11eb-908f-30a2e9aecc2b.png)

**Comment by dutyhong** [comment]
i have the same problem

**Comment by Vydruth** [comment]
I agree. My team would need this feature as well. This might be loosely related to: https://github.com/doccano/doccano/discussions/1304
